src/scenes/home/gala/gala.js

```diff
@@ -1,119 +1,58 @@
-            <li>Featured in all event promotional materials</li>
-            <li>Logo featured on home page of Operation Code website</li>
-            <li>One table at the Gala with prime VIP seating for 8</li>
-            <li>Full page color ad in the program and acknowledgement in event remarks</li>
-            <li>Sponsorship of one veteran to a tech conference including travel and lodging</li>
-            <li>Sponsorship of two veterans to Operation Code meetups around the United States</li>
-            <li>Recognition on Operation Code social media</li>
-            <li>Blog interview on your company’s involvement with Operation Code</li>
-          </ul>
-        </div>
-        <div>
-          <h3>Four Star Sponsor $10,000</h3>
-          <ul>
-            <li>Featured on the inside back cover of the Deploy the Future Gala event catalog</li>
-            <li>Logo Featured on home page of Operation Code website</li>
-            <li>One table at the Gala with prime VIP seating for 8</li>
-            <li>3/4 page color ad in program and acknowledgement in event remarks</li>
-            <li>Sponsorship of two veterans to Operation Code meetups around the United States</li>
-            <li>Recognition on Operation Code social media</li>
-            <li>Blog interview on your company’s involvement with Operation Code</li>
-          </ul>
-        </div>
-        <div>
-          <h3>Three Star Sponsor $5,000</h3>
-          <ul>
-            <li>Priority seating for 8</li>
-            <li>1/2 page color ad in the event program</li>
-            <li>Special recognition in print and web materials</li>
-            <li>Sponsorship of one veteran to an Operation Code meetup around the United States</li>
-            <li>Blog interview on your company’s involvement with Operation Code</li>
-          </ul>
-        </div>
-        <div>
-          <h3>Scholar Host $2,500</h3>
-          We know not all of our supporters can make the trip to Portland for the Gala but you can still help by sponsoring the travel and lodging for an Operation Code Veteran Scholar so they can attend Deploy the Future and share their story with our audience. This will provide the scholarship recipient with an incredible networking opportunity. You will be making a tremendous impact on this person’s future.
-        </div>
-        <div>
-          <h3>Specialty Patron of Technology $2,500</h3>
-          This is an opportunity for an Operation Code supporter to underwrite the technology needs for the Deploy the Future Gala. Operation Code knows the importance of using the right software for the right ask and that is why we are using Greater Giving Online/Mobile Bidding software at our first annual gala.
-        </div>
-        <div>
-          <h3>Patron $1250</h3>
-          <ul>
-            <li>Table seating for 8</li>
-          </ul>
+          <p>Check back for more information</p>
```

Comment:
Perhaps add a period at the end of this sentence, so it matches the 'Contact us for sponsorship information.' sentence above.